.gitlab/pipeline/test.yml

```diff
@@ -29,7 +29,7 @@ test-linux-stable:
         --locked \
         --release \
         --no-fail-fast \
-        --features try-runtime,experimental \
+        --features try-runtime,experimental,ci-only-tests \
```

Comment:
Pretty sure this used to be here!

Reply:
@paritytech/ci How can we remove this feature for `test-linux-oldkernel-stable`? It's expected that this test fails on old kernels.

Reply:
considering that the feature is called `ci-only-tests` and not smth like `ci-only-tests-new-kernel`, maybe we could instead check the kernel version in the test (using `uname`) and skip the test if it's too old

Reply:
Great idea @alindima, done.